cfy_manager/components/composer/composer.py

```diff
@@ -188,3 +188,8 @@ def remove(self):
         logger.notice('Removing Composer data....')
         common.sudo(['rm', '-rf', '/opt/cloudify-composer'])
         logger.notice('Cloudify Composer successfully removed')
+
+    def upgrade(self):
+        logger.notice('Upgrading Cloudify Composer...')
+        self._run_db_migrate()
```

Comment:
This function is identical to the one in the stage component. Should we run them both during the upgrade?

Reply:
But `self`so we'd be executing two different functions: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/187d097bd3fb3007e24a3a4d3e0a802603902de6/cfy_manager/components/composer/composer.py#L64 and https://github.com/cloudify-cosmo/cloudify-manager-install/blob/187d097bd3fb3007e24a3a4d3e0a802603902de6/cfy_manager/components/stage/stage.py#L74.

OFC we could add one more level of abstraction (say `UiComponent` such as: `BaseComponent` -> `UiComponent` -> `{Composer,Stage}Component`) but that's probably an overkill for now.